serde_derive/src/internals/attr.rs

```diff
@@ -345,6 +347,16 @@ impl Container {
                     }
                 }
 
+                // Parse `#[serde(alias_all = "foo")]`
+                Meta(NameValue(m)) if m.path == ALIAS_ALL => {
+                    if let Ok(s) = get_lit_str(cx, ALIAS_ALL, &m.lit) {
```

Comment:
PascalCase and camelCase differs only by the first character, maybe we can have an optimization to not check all the characters twice?

Reply:
I am not sure to understand, the renaming is done somewhere else and is not impacted by this change. I think this optimization is not needed and would be hard to implement given current implementation.